src/locales/en_US.ts

```diff
@@ -1314,6 +1314,13 @@ const strings = {
 
   // #endregion GuiPlugins
 
+  // #region Light Account
+
+  light_account_card_backup_title: 'Back up your account',
```

Comment:
Rename key to `backup_account_title` to make more generic

Reply:
The suggestion seems more specific rather than generic. 
"Backup Account" describes a subset (modal, reduced signup flow) of the overall "Light Account" feature